srcpkgs/libratbag/template

```diff
@@ -9,13 +9,15 @@ hostmakedepends="pkg-config git swig"
 makedepends="elogind-devel eudev-libudev-devel libglib-devel libevdev-devel
  python3-devel json-glib-devel libunistring-devel"
 depends="dbus python3-evdev python3-gobject"
-checkdepends="python3-gobject python3-evdev"
+checkdepends="dbus python3-gobject python3-evdev"
 short_desc="DBus daemon to configure gaming mice"
 maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="MIT"
 homepage="https://github.com/libratbag/libratbag"
 distfiles="https://github.com/libratbag/libratbag/archive/v${version}.tar.gz"
-checksum=9e6ad8d54daef8cc0f44784bdde9bf5a3879d8a66d11e0b990b87266f81f7329
+checksum=c70b6a259693216e6a57ec3087496e8d6df01d4425b7a06592872b7b123e5fcc
+make_check="ci-skip" # needs system dbus
```

Comment:
```suggestion
make_check=extended # needs system dbus
```
ok, this should be the final issue.

Thanks for the prompt changes :)

Reply:
from Manual.md:

> - `make_check` Sets the cases in which the `check` phase is run.
> This option has to be accompanied by a comment explaining why the tests fail.
> Allowed values:
>   - `yes` (the default) to run if `XBPS_CHECK_PKGS` is set.
>   - `extended` to run if `XBPS_CHECK_PKGS` is `full`.
>   - `ci-skip` to run locally if `XBPS_CHECK_PKGS` is set, but not as part of pull request checks.
>   - `no` to never run.

you can set `XBPS_CHECK_PKGS` in `etc/conf`. Extended is for when checks requires some specific system setup, in this case, the system dbus service being enabled.